entrypoint.sh

```diff
@@ -1,23 +1,26 @@
-#!/bin/sh
```

Comment:
We either need to remove it or change it to /bin/bash. Unlike most systems, in a GH runner, `sh` is not just an alias for `bash`. I opted to remove it entirely to avoid confusion and have the runner dictate the behaviour.

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell